regex/processors/context.go

```diff
@@ -7,16 +7,19 @@ import (
 	"fmt"
 	"io"
 	"os"
+
+	"github.com/theseion/crs-toolchain/v2/context"
 )
 
 type Context struct {
-	rootDirectory      string
-	rulesDirectory     string
-	utilDirectory      string
-	dataFilesDirectory string
-	includeFiles       string
-	singleRuleID       int
-	singleChainOffset  bool
+	rootDirectory         string
+	rulesDirectory        string
+	utilDirectory         string
+	dataFilesDirectory    string
+	includeFilesDirectory string
+	includeFiles          string
+	singleRuleID          int
+	singleChainOffset     bool
 }
```

Comment:
Do we need two different contexts?

Reply:
I need the directories for other things, but not the rest. That's why I moved them to a separate context. Not too happy with how it turned out... I was hoping that Karel would clean that up later 😜 .

Reply:
But it is confusing still. Why importing `"github.com/theseion/crs-toolchain/v2/context"` then? Maybe you want to extend it?

Reply:
Do you like this version better? I do like it a little bit better.